docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process where all required resources and dependencies are declared and prefetched within a build system. This strict control ensures that the build system has complete visibility and control over the dependencies needed for the build, including the ability to accurately collect, inquire, analyze, and manage all dependencies during the build process. By restricting external dependencies and implementing a centralized management approach, hermetic builds provide a predictable and reliable build environment.
```

Comment:
Are you trying to be general here or specific to {ProductName}? If being generic, I don't think the first sentence is quite accurate, instead something like this may be better

```suggestion
Hermetic builds refer to a controlled build process where all required resources and dependencies are completely known by a build system. This dependency visibility ensures that the build system can accurately collect, inquire, analyze, and manage all dependencies during the build process.
In {ProductName}, hermetic builds are achieved by blocking network access to the build itself, so all dependencies must be pre-fetched in order to be available at build time.
```

I just removed the last sentence. If you like it, that may fit better elsewhere. I also introduced network isolation as you just state that it is happening later without connecting it to hermetic builds.